docs/index.yaml

```diff
@@ -3,7 +3,27 @@ entries:
   keda:
   - apiVersion: v1
     appVersion: 1.4.1
```

Comment:
`appVersion: 1.4.2` ?

Reply:
That's actually a fix for https://github.com/kedacore/charts/pull/52 since we don't have 1.4.2 (https://github.com/kedacore/keda/releases)

Reply:
Gotcha, it's Monday, sorry 😄